app.js

```diff
@@ -50,36 +56,48 @@ function showButton() {
     submitButton.classList.add('show-button');
 }
 
-function startPlaying() {
-    
+function startPlaying() {     
+    if(randomNumber === 0) {
+        appState.computerSelectedItem = paper;
+    } else if (randomNumber === 1) {
+        appState.computerSelectedItem = rock;
+    } else {
+        appState.computerSelectedItem = scissors;
+    }
+}
+
+function getRandom(min, max) {
```

Comment:
when `getRandom` is invoked, there is no arguments passed - you can remove arguments definition from parenthesis.